### PR TITLE
Add extra info on courses page

### PR DESCRIPTION
**Pull request by triszt4n**
Just some extra info on courses page

### CODE REVIEW

File: src/pages/courses.tsx [hunk 1]
```diff
@@ -83,13 +83,30 @@ const CoursesPage: React.FC<CoursesProps> = ({ data }) => (
...
+          </Heading>
+          <Text mt={4}>
+            Zúg a projektor, magyaráz az előadó. Nincs szükséged nagy előtudásra, ha tudod, mik azok az alapvető adatstruktúrák és hogy kell
+            elágazásokat írni, már jó vagy. Kellemes, baráti atmoszféra, gyakori szakmai poénok és segítőkész segédelőadók várnak. Az, hogy
+            mit kell hozni, hova kell menni, minden tanfolyamnál különböző, e-mailben fogod megkapni az összes tudnivalót időben az
+            előadások előtt.
+          </Text>
+          <Flex mt={10} direction={{ base: 'column', md: 'row' }}>
+            <Heading as="h3" size="md">
+              Sajnos a férőhelyek száma korlátos, így a tanfolyam résztvevőinek listáját jelentkezési sorrend alapján alakítjuk ki.
```
Comment (OmTheTurtle) [inline src/pages/courses.tsx:108]:
```suggestion
              Sajnos a férőhelyek száma korlátozott, így a tanfolyam résztvevőinek listáját jelentkezési sorrend alapján alakítjuk ki.
```

File: src/pages/courses.tsx [hunk 2]
```diff
@@ -83,13 +83,30 @@ const CoursesPage: React.FC<CoursesProps> = ({ data }) => (
...
-              férőhelyek száma korlátos, így a tanfolyam résztvevőinek listáját jelentkezési sorrend alapján alakítjuk ki.
-            </Text>
+          <Heading as="h3" size="md" mt={4}>
+            Miért érdemes jelentkezned tanfolyamra?
+          </Heading>
+          <Text mt={4}>
+            Ha az utóbbi pár félévben úgy érezted, hogy szeretsz új programnyelveket megismerni, és azokat valódi fejlesztési környezetekben
+            szeretnéd kipróbálni, itt a helyed. Ha szeretnél elsajátítani olyan tudást, ami az iparban is haszon lehet, itt a helyed. Ha
+            több ezres tömegeket elérő alkalmazásokat szeretnél fejleszteni, szeretnéd megmutatni a tervezési képességeid, csapatban
+            szeretnél tanulni és munkálkodni, itt a helyed.
```
Comment (OmTheTurtle) [inline src/pages/courses.tsx:95]:
```suggestion
            Körünkben lehetőséged van különböző webes technológiák megismerésére, amelyek használatát valódi, esetenként több ezres tömegeket kiszolgáló alkalmazások fejlesztésén keresztül sajátíthatsz el.
Nálunk olyan tudásra tehetsz szert, amit később közvetlenül az iparban is hasznosíthatsz. Szeretünk csapatban dolgozni, így fontosnak tartjuk az ezzel kapcsolatos soft-skillek fejlesztését is.
```

File: src/pages/courses.tsx [hunk 3]
```diff
@@ -83,13 +83,30 @@ const CoursesPage: React.FC<CoursesProps> = ({ data }) => (
...
+          </Text>
+          <Heading as="h3" size="md" mt={4}>
+            Mire számíthatsz a tanfolyamokon?
+          </Heading>
+          <Text mt={4}>
+            Zúg a projektor, magyaráz az előadó. Nincs szükséged nagy előtudásra, ha tudod, mik azok az alapvető adatstruktúrák és hogy kell
+            elágazásokat írni, már jó vagy. Kellemes, baráti atmoszféra, gyakori szakmai poénok és segítőkész segédelőadók várnak. Az, hogy
+            mit kell hozni, hova kell menni, minden tanfolyamnál különböző, e-mailben fogod megkapni az összes tudnivalót időben az
+            előadások előtt.
```
Comment (OmTheTurtle) [inline src/pages/courses.tsx:104]:
```suggestion
            Tanfolyamainkon lehetőséged van betekinteni az általunk használt technológiákba és projektjeinkbe. Kellemes, baráti atmoszféra, gyakori szakmai poénok és segítőkész segédelőadók várnak. Nálunk nincsenek szigorú előkövetelmények, ha tudod, mik azok az alapvető adatstruktúrák és hogy kell elágazásokat írni, már jó vagy. Az egyes alkalmakról minden tudnivalót (előadás témája, mit kell hozni, hova kell menni) előzetesen email-ben fogsz megkapni az előadás előtt.
```

File: src/components/course-components/CourseCard.tsx [hunk 4]
```diff
@@ -32,7 +32,7 @@ const getSessionString = (session: ISession): string => {
 
 const CourseCard: React.FC<CourseCardProps> = ({ course }) => (
   <Box borderWidth="1px" rounded="lg" shadow="lg">
-    <Box pt={4} pb={6} px={4}>
+    <Box pt={4} pb={4} px={4}>
```
Comment (triszt4n) [inline src/components/course-components/CourseCard.tsx:35]: This is due to the new markdown.css paddings

File: src/components/course-components/CourseCard.tsx [hunk 5]
```diff
@@ -57,7 +57,7 @@ const CourseCard: React.FC<CourseCardProps> = ({ course }) => (
           </HStack>
         )}
       </Box>
-      <Box mt={6} fontSize="md">
+      <Box mt={2} fontSize="md">
```
Comment (triszt4n) [inline src/components/course-components/CourseCard.tsx:60]: This too

**Review by triszt4n — COMMENTED** [review]
Basically copied all your suggestions, @OmTheTurtle. They were nicely written.

**Review by OmTheTurtle — APPROVED** [review]

**Pull request merged**